app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ArticlesController < ApplicationController
   before_action :set_article, only: [:show, :edit, :update]
 
+  def index
+    @articles = current_user.articles.all
```

Comment:
**suggestion (non-blocking):** Add some ordering.

What order do we expect to display these articles? There is an implied ordering, but it pays to make these things explicit.

Reply:
Addressed in PR #49